File: app/src/main/java/com/creat/motiv/NewUser.java
package com.creat.motiv;

import android.content.Intent;
import android.graphics.Color;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import androidx.appcompat.app.AppCompatActivity;
import androidx.viewpager.widget.ViewPager;

import com.creat.motiv.Adapters.ViewPagerAdapter;
import com.google.android.material.tabs.TabLayout;

import java.util.Objects;

public class NewUser extends AppCompatActivity {
    ViewPagerAdapter adapter;
    TabLayout tabs;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_new_user);
        final Button start = findViewById(R.id.start);
          tabs = findViewById(R.id.tabs);
        ViewPager slides = findViewById(R.id.slides);
        tabs.setupWithViewPager(slides,true);
        adapter = new ViewPagerAdapter(this, this);
        slides.setAdapter(adapter);
        setupTabIcons();
         start.setVisibility(View.INVISIBLE);
        start.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Comecar();
            }
        });


        slides.addOnPageChangeListener(new ViewPager.OnPageChangeListener() {
            @Override
            public void onPageScrolled(int position, float positionOffset, int positionOffsetPixels) {
                if (position == 7 ){
                    start.setVisibility(View.VISIBLE);
                }
            }

            @Override
            public void onPageSelected(int position) {
                if (position == 7 ){
                    start.setVisibility(View.VISIBLE);
                }
            }

            @Override
            public void onPageScrollStateChanged(int state) {

            }
        });

    }
    private void setupTabIcons() {
        tabs.setSelectedTabIndicatorColor(Color.TRANSPARENT);

        for (int i = 0; i  < 7; i++){
            Objects.requireNonNull(tabs.getTabAt(i)).setIcon(R.drawable.dot);
        }

    }
    private void Comecar(){

        Intent i = new Intent(this,MainActivity.class);
        i.putExtra("novo",true);
        startActivity(i);
        this.finish();

    }
}

File: app/src/main/java/com/creat/motiv/view/activities/NewUser.kt
package com.creat.motiv.View.activities

import android.content.Intent
import android.graphics.Color
import android.os.Bundle
import android.view.View
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity
import androidx.viewpager.widget.ViewPager
import com.creat.motiv.R
import com.creat.motiv.adapters.ViewPagerAdapter
import com.creat.motiv.view.activities.MainActivity
import com.google.android.material.tabs.TabLayout

class NewUser : AppCompatActivity() {
    var adapter: ViewPagerAdapter?? = null
    var tabs: TabLayout?? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_new_user)
        val start = findViewById<Button>(R.id.start)
        tabs = findViewById(R.id.tabs)
        val slides = findViewById<ViewPager>(R.id.slides)
        tabs!!.setupWithViewPager(slides, true)
        adapter = ViewPagerAdapter(this, this)
        slides.adapter = adapter
        setupTabIcons()
        start.visibility = View.INVISIBLE
        start.setOnClickListener { Comecar() }


        slides.addOnPageChangeListener(object : ViewPager.OnPageChangeListener {
            override fun onPageScrolled(position: Int, positionOffset: Float, positionOffsetPixels: Int) {
                if (position == 7) {
                    start.visibility = View.VISIBLE
                }
            }

            override fun onPageSelected(position: Int) {
                if (position == 7) {
                    start.visibility = View.VISIBLE
                }
            }

            override fun onPageScrollStateChanged(state: Int) {

            }
        })

    }

    private fun setupTabIcons() {
        tabs!!.setSelectedTabIndicatorColor(Color.TRANSPARENT)

        for (i in 0..6) {
            tabs!!.getTabAt(i)!!.icon = getDrawable(R.drawable.dot)
        }

    }

    private fun Comecar() {

        val i = Intent(this, MainActivity::class.java)
        i.putExtra("novo", true)
        startActivity(i)
        this.finish()

    }
}
